# Profitable App Profiles from Google Play Store and Apple Store Market
---
In this project, we will evaluate different apps in both the Google Play Store and the Apple Store so that we can gain some insight into which apps will produce more revenue for our company based on our development criteria.

My aim in this project is to find mobile profiles that are profitable for the Google Play Store and the Apple Store markets. This will help our developers understand what the market demands and how we will be able to fulfill the need in order to be successful.

At our company, we will create apps that are free to download and install, and our key revenue stream consists of in-app advertising. This means that our revenue for any given app is largely determined by the number of users using the app. Our aim for this project is to analyze data to help our developers understand what kind of apps are likely to attract more users.

## The Data Opening and Exploring

Approximately 2 million iOS apps were available on the App Store and 2.1 million Android apps on Google Play as of September 2018.

Collecting data for more than 4 million apps requires a considerable amount of time and money, so we'll try to analyze a sample of data instead. In order to avoid spending resources on collecting new data on our own, we should first try to see if we can find any relevant existing data at no cost. Luckily, these are two data sets that seem to be suited to our objectives:

- [Google Play Store](https://www.kaggle.com/lava18/google-play-store-apps) A data set containing approximately 10,000 Android apps from Google Play; the data was collected in August 2018. The data set can be downloaded directly from the link.

- [Apple Store](https://www.kaggle.com/ramamet4/app-store-apple-data-set-10k-apps) Data set containing approximately 7,000 iOS apps from the App Store; the data was collected in July 2017. The data set can be downloaded directly from the link.

Let's start by opening the two data sets and then continue to explore the data.

In [1]:
# We will first need to import the csv files into our Jupyter notebook

### The Apple Store data set ###
apple_file = open('AppleStore.csv')
from csv import reader
read_applefile = reader(apple_file)
apple_apps = list(read_applefile)

### The Google Play data set ###
google_file = open('googleplaystore.csv')
read_googlefile = reader(google_file)
google_apps = list(read_googlefile)



---

_To make it easier to explore the two data sets, we will first write a function called `explore_data()` which we can use repeatedly to explore the rows in a more readable way. We will also add an option for our function to show the number of rows and columns for any data set._

---

In [2]:
# We are going to now create a function so we can quickly explore any data set

def explore_data(dataset, start, end, rows_and_columns = False):
    dataset_slice = dataset[start:end]
    for row in dataset_slice:
        print(row)
        print('\n') # Adds a new empty line after each row
        
    if rows_and_columns:
        print('Number of rows:', len(dataset))
        print('Number of columns:', len(dataset[0]))
        

In [3]:
## We are going to test the functions 

print(apple_apps[0])
print('\n')
explore_data(apple_apps[1:],0,3,True)
print('\n')
print(google_apps[0])
print('\n')
explore_data(google_apps[1:],0,3,True)

['id', 'track_name', 'size_bytes', 'currency', 'price', 'rating_count_tot', 'rating_count_ver', 'user_rating', 'user_rating_ver', 'ver', 'cont_rating', 'prime_genre', 'sup_devices.num', 'ipadSc_urls.num', 'lang.num', 'vpp_lic']


['284882215', 'Facebook', '389879808', 'USD', '0.0', '2974676', '212', '3.5', '3.5', '95.0', '4+', 'Social Networking', '37', '1', '29', '1']


['389801252', 'Instagram', '113954816', 'USD', '0.0', '2161558', '1289', '4.5', '4.0', '10.23', '12+', 'Photo & Video', '37', '0', '29', '1']


['529479190', 'Clash of Clans', '116476928', 'USD', '0.0', '2130805', '579', '4.5', '4.5', '9.24.12', '9+', 'Games', '38', '5', '18', '1']


Number of rows: 7197
Number of columns: 16


['App', 'Category', 'Rating', 'Reviews', 'Size', 'Installs', 'Type', 'Price', 'Content Rating', 'Genres', 'Last Updated', 'Current Ver', 'Android Ver']


['Photo Editor & Candy Camera & Grid & ScrapBook', 'ART_AND_DESIGN', '4.1', '159', '19M', '10,000+', 'Free', '0', 'Everyone', 'Art & Design', 

>**_Now we're going to look at the names of the columns(fields) to see which ones are going to be useful in our study. To grasp an understanding of the columns and what they contain we will look at the documentations._** 
[GooglePlayStore](https://www.kaggle.com/lava18/google-play-store-apps)  [AppleStore](https://www.kaggle.com/ramamet4/app-store-apple-data-set-10k-apps) 

_At a quick glance, from the Google Play Store the columns that might be useful for the purpose of our analysis are: 'App', 'Category', 'Review', 'Installs', 'Type', 'Price', and 'Genres._ 

_From the Apple Store the columns that seem interesting are: 'track_name', 'currency', 'price', 'rating_count_tot', 'rating_count_ver', and 'prime_genre'._

In [4]:
print(apple_apps[0])
print('\n')
print(google_apps[0])

['id', 'track_name', 'size_bytes', 'currency', 'price', 'rating_count_tot', 'rating_count_ver', 'user_rating', 'user_rating_ver', 'ver', 'cont_rating', 'prime_genre', 'sup_devices.num', 'ipadSc_urls.num', 'lang.num', 'vpp_lic']


['App', 'Category', 'Rating', 'Reviews', 'Size', 'Installs', 'Type', 'Price', 'Content Rating', 'Genres', 'Last Updated', 'Current Ver', 'Android Ver']


## Removing Inaccurate Data

> **_Now before we begin our analysis, we need to make sure the data we analyze is accurate, otherwise the results of our analysis will be false. This means that we need to:_**
 - Detect inaccurate data, and correct or remove it.
 - Detect duplicate data, and remove the duplicates. 

_The Google Play data set has a dedicated [discussion section](https://www.kaggle.com/lava18/google-play-store-apps/discussion) and we can see that one of the discussions describes an error for a certain row. We are going to read the discussion section for the App Store data set, and see whether we can find any report of wrong data in our dataset._

In [5]:
print(google_apps[0])
print('\n')

# We found the index in the discussion section and we are going to test it 
print(google_apps[10472])
print('\n')

# Incase they included the header or not 
print(google_apps[10473])

['App', 'Category', 'Rating', 'Reviews', 'Size', 'Installs', 'Type', 'Price', 'Content Rating', 'Genres', 'Last Updated', 'Current Ver', 'Android Ver']


['Xposed Wi-Fi-Pwd', 'PERSONALIZATION', '3.5', '1042', '404k', '100,000+', 'Free', '0', 'Everyone', 'Personalization', 'August 5, 2014', '3.0.0', '4.0.3 and up']


['Life Made WI-Fi Touchscreen Photo Frame', '1.9', '19', '3.0M', '1,000+', 'Free', '0', 'Everyone', '', 'February 11, 2018', '1.0.19', '4.0 and up']


---
The row 10473 (with headers) corresponds to the app **_Life Made Wi-Fi Touchscreen Photo Frame_** , and we can see that the rating is 19. This is clearly off because the maximum rating for a Google Play app is 5. Hence we will delete this row.

--- 

In [6]:
# We found the row with the wrong data and now we are going to delete it 
del google_apps[10473]

## Removing Duplicate Entries 

### Part One

_Now we are going to check for duplicates in the Google Play Store data set_

In [7]:
duplicate_list = []
unique_list = []

for app in google_apps[1:]:
    name = app[0]
    if name in unique_list:
        duplicate_list.append(name)
    else:
        unique_list.append(name)
        
print(duplicate_list[:3])

['Quick PDF Scanner + OCR FREE', 'Box', 'Google My Business']


---

_The above list shows a few examples of duplicate entries in the data set. Let's see just how many duplicates there are._

---

In [8]:
print('Number of duplicates are:', len(duplicate_list))

Number of duplicates are: 1181


---

_Our codes show us that we have 1181 duplicates and we're going to have to remove them. However, we're not going to remove duplicates at random. We're going to keep the one with the highest number of reviews seeing that it's the most recent and get rid of the rest._

---

In [9]:
# We are going to find out what our expected number of apps should be when we delete the duplicates
print('Expected number of apps are:', len(google_apps[1:])- len(duplicate_list))

Expected number of apps are: 9659


In [10]:
# Now we are going to start the process of deleting the duplicates and keeping the ones with the highest reviews
reviews_max  = {}
for app in google_apps[1:]:
    name = app[0]
    n_reviews = float(app[3])
    if name in reviews_max and reviews_max[name] < n_reviews:
        reviews_max[name] = n_reviews
    elif name not in reviews_max:
        reviews_max[name] = n_reviews

print('Actual number of apps are: ',len(reviews_max))

Actual number of apps are:  9659


---

*In the code above, we list all the names of the apps and their highest reviews. We're going to use this to delete all the duplicates.*


### Part Two

_Let's use the reviews_max dictionary to remove the duplicates. For the duplicate cases, we'll only keep the entries with the highest number of reviews. In the code cell below:_

- We start by initializing two empty lists, android_clean and already_added.
- We loop through the android data set, and for every iteration:
    - We isolate the name of the app and the number of reviews.
    - We add the current row (app) to the android_clean list, and the app name (name) to the already_added list if:
        The number of reviews of the current app matches the number of reviews of that app as described in the reviews_max dictionary; and
    The name of the app is not already in the already_added list. We need to add this supplementary condition to account for those cases where the highest number of reviews of a duplicate app is the same for more than one entry (for example, the Box app has three entries, and the number of reviews is the same). If we just check for reviews_max[name] == n_reviews, we'll still end up with duplicate entries for some apps.

---

In [11]:
android_clean = []
already_added = []
for app in google_apps[1:]:
    name = app[0]
    n_reviews = float(app[3])
    if (n_reviews == reviews_max[name]) and (name not in already_added):
        android_clean.append(app)
        already_added.append(name)
        
len(android_clean)

9659

## Removing Non-English Apps

### Part One

*Now that we've got our clean data set, we're going to make sure that all apps are English-speaking apps since we're catering to an English audience.
We will create a function that will help us quickly identify English versus non-English apps.*

In [12]:
def english_check(string):
    count = 0
    for i in string:
        if ord(i) > 127: # ASCII standard, characters for English apps fall between 0 and 127
            count += 1
           
    if count > 3:
        return False
    else:
        return True
            

In [13]:
# We are going to test our function with a few examples
print(english_check('Instagram'))
print(english_check('爱奇艺PPS -《欢乐颂2》电视剧热播'))
print(english_check('Docs To Go™ Free Office Suite'))
print(english_check('Instachat 😜'))

True
False
True
True


---

### Part Two

_Since we've created our function to help us quickly identify English apps, we're now going to use it to further clean up our list._


In [14]:
english_android_clean = []
for app in android_clean:
    name = app[0]
    if english_check(name):
        english_android_clean.append(app)
        
english_apple_apps = []
for app in apple_apps[1:]:
    name = app[1]
    if english_check(name):
        english_apple_apps.append(app)
        

In [15]:
# Check how many English apps are actually in the data set
print(len(english_android_clean))
print(len(english_apple_apps))

9614
6183


---

## Isolating the Free Apps

*So far in the data cleaning process, we:*

- Removed inaccurate data
- Removed duplicate app entries
- Removed non-English apps

*Now we are going to isolate all the free apps since we are only interested in those.*

In [16]:
free_apple_apps = []
for app in english_apple_apps:
    price = float(app[4])
    if price == 0.0:
        free_apple_apps.append(app)

free_google_apps = []
check =[]
for app in english_android_clean:
    price = app[7]
    if price[0] == '$': # Some of the price have a $ infront of the number so we have to remove it
        price = price[1:]
    if float(price) == 0.0:
        free_google_apps.append(app)
        
print("Number of free Apple apps are: ", len(free_apple_apps))
print('\n')
print('Number of free Google apps are: ', len(free_google_apps))

Number of free Apple apps are:  3222


Number of free Google apps are:  8864


## Most Common Apps by Genre

> **So far, we spent a good amount of time on cleaning data, and:**

> - Removed inaccurate data
- Removed duplicate app entries
- Removed non-English apps
- Isolated the free apps

> **As we mentioned before, our aim is to determine the kinds of apps that are likely to attract more users because our revenue is highly influenced by the number of people using our apps.**

> **To minimize risks and overhead, our validation strategy for an app idea is comprised of three steps:**

> - Build a minimal Android version of the app, and add it to Google Play.
- If the app has a good response from users, we develop it further.
- If the app is profitable after six months, we build an iOS version of the app and add it to the App Store.
  
> **Because our end goal is to add the app on both Google Play and the App Store, we need to find app profiles that are successful on both markets.** 

In [17]:
# We are going to build two functions to help us identify apps that are successful on both Google Play and Apple Store 

def freq_table(dataset, index):
    freq = {}
    for row in dataset:
        if row[index] in freq:
            freq[row[index]] += 1
        else:
            freq[row[index]] = 1
    for key in freq:
        freq[key] = round((freq[key]/len(dataset)) * 100,2) # turn the value into percentage
    return freq
    
def display_table(dataset, index):
    table = freq_table(dataset, index)
    table_display = []
    for key in table:
        key_val_as_tuple = (table[key], key)
        table_display.append(key_val_as_tuple)
    
    table_sorted = sorted(table_display, reverse = True)
    for entry in table_sorted:
        print(entry[1], ' : ', entry[0])
    

In [18]:
# Test the function on the Apple Apps 
display_table(free_apple_apps, 11)

Games  :  58.16
Entertainment  :  7.88
Photo & Video  :  4.97
Education  :  3.66
Social Networking  :  3.29
Shopping  :  2.61
Utilities  :  2.51
Sports  :  2.14
Music  :  2.05
Health & Fitness  :  2.02
Productivity  :  1.74
Lifestyle  :  1.58
News  :  1.33
Travel  :  1.24
Finance  :  1.12
Weather  :  0.87
Food & Drink  :  0.81
Reference  :  0.56
Business  :  0.53
Book  :  0.43
Navigation  :  0.19
Medical  :  0.19
Catalogs  :  0.12


---

We can see that more than half (58.16 percent) of the free English apps are games. Entertainment apps are close to 8 percent, followed by photo and video apps close to 5 percent. Only 3.66 per cent of apps are designed for education, followed by social networking apps, which represent 3.29 per cent of apps in our data set.

The general impression is that Apple Store (at least part containing free English apps) is dominated by fun-oriented apps (games, entertainment, photos and videos, social networking, sports, music, etc.), while practical apps (education, shopping, utilities, productivity, lifestyle, etc.) are more rare. However, the fact that fun apps are the most numerous does not imply that they also have the largest number of users- the demand may not be the same as the offer.

Let us continue by looking at the Genres and Categories columns of the Google Play data set (two columns that appear to be related).

---


In [19]:
# Test the function on Google Apps, showing the Category columns by percentage make up
display_table(free_google_apps,1)

FAMILY  :  18.91
GAME  :  9.72
TOOLS  :  8.46
BUSINESS  :  4.59
LIFESTYLE  :  3.9
PRODUCTIVITY  :  3.89
FINANCE  :  3.7
MEDICAL  :  3.53
SPORTS  :  3.4
PERSONALIZATION  :  3.32
COMMUNICATION  :  3.24
HEALTH_AND_FITNESS  :  3.08
PHOTOGRAPHY  :  2.94
NEWS_AND_MAGAZINES  :  2.8
SOCIAL  :  2.66
TRAVEL_AND_LOCAL  :  2.34
SHOPPING  :  2.25
BOOKS_AND_REFERENCE  :  2.14
DATING  :  1.86
VIDEO_PLAYERS  :  1.79
MAPS_AND_NAVIGATION  :  1.4
FOOD_AND_DRINK  :  1.24
EDUCATION  :  1.16
ENTERTAINMENT  :  0.96
LIBRARIES_AND_DEMO  :  0.94
AUTO_AND_VEHICLES  :  0.93
HOUSE_AND_HOME  :  0.82
WEATHER  :  0.8
EVENTS  :  0.71
PARENTING  :  0.65
ART_AND_DESIGN  :  0.64
COMICS  :  0.62
BEAUTY  :  0.6


In [20]:
# Test the function on Google Apps
display_table(free_google_apps, 9)

Tools  :  8.45
Entertainment  :  6.07
Education  :  5.35
Business  :  4.59
Productivity  :  3.89
Lifestyle  :  3.89
Finance  :  3.7
Medical  :  3.53
Sports  :  3.46
Personalization  :  3.32
Communication  :  3.24
Action  :  3.1
Health & Fitness  :  3.08
Photography  :  2.94
News & Magazines  :  2.8
Social  :  2.66
Travel & Local  :  2.32
Shopping  :  2.25
Books & Reference  :  2.14
Simulation  :  2.04
Dating  :  1.86
Arcade  :  1.85
Video Players & Editors  :  1.77
Casual  :  1.76
Maps & Navigation  :  1.4
Food & Drink  :  1.24
Puzzle  :  1.13
Racing  :  0.99
Role Playing  :  0.94
Libraries & Demo  :  0.94
Auto & Vehicles  :  0.93
Strategy  :  0.91
House & Home  :  0.82
Weather  :  0.8
Events  :  0.71
Adventure  :  0.68
Comics  :  0.61
Beauty  :  0.6
Art & Design  :  0.6
Parenting  :  0.5
Card  :  0.45
Casino  :  0.43
Trivia  :  0.42
Educational;Education  :  0.39
Board  :  0.38
Educational  :  0.37
Education;Education  :  0.34
Word  :  0.26
Casual;Pretend Play  :  0.24
Music  :  0.2
R

---

The difference between the Genres and the Category columns is not crystal clear, but one thing we can see is that the Genres column is much more granular (it has more categories). We're just looking for a larger picture at the moment, so we're just going to work with the Category column moving forward. 

Up to this point, we've found that the Apple Store is dominated by fun apps, while Google Play shows a more balanced picture of both practical and fun apps. Now we'd like to get an idea of the kind of apps most users have.

## Most Popular Genre Apps Available on the Apple Store

We now know the most popular apps in each dataset, but to really know if those apps will generate revenue for our company, we'll need to look at the user rating or the apps installment to understand their traction level. One way to find out what genres the most users have is to calculate the average number of installs for each genre of app. This information can be found in the `Installs` column for the Google Play data set, but this information is missing for the App Store data set. As a workaround, we'll take the total number of user ratings as a proxy that can be found in the `rating_count_tot` column.

Below, we calculate the average number of ratings per app genre in the App Store:

In [21]:
# We are going to generate a frequency table for the Apple Store Genre column 
freq_table(free_apple_apps,11)

{'Social Networking': 3.29,
 'Photo & Video': 4.97,
 'Games': 58.16,
 'Music': 2.05,
 'Reference': 0.56,
 'Health & Fitness': 2.02,
 'Weather': 0.87,
 'Utilities': 2.51,
 'Travel': 1.24,
 'Shopping': 2.61,
 'News': 1.33,
 'Navigation': 0.19,
 'Lifestyle': 1.58,
 'Entertainment': 7.88,
 'Food & Drink': 0.81,
 'Sports': 2.14,
 'Book': 0.43,
 'Finance': 1.12,
 'Education': 3.66,
 'Productivity': 1.74,
 'Business': 0.53,
 'Catalogs': 0.12,
 'Medical': 0.19}

In [22]:
# We are going to calculate the average user rating  for each genre in Apple Store data set 
for key in freq_table(free_apple_apps,11):
    total = 0
    len_genre = 0
    for i in free_apple_apps:
        genre_app = i[11]
        if genre_app == key:
            num_rating = float(i[5])
            total += num_rating
            len_genre += 1
    avg_user_rating = total / len_genre
    print(key, " : ", round(avg_user_rating))

Social Networking  :  71548
Photo & Video  :  28442
Games  :  22789
Music  :  57327
Reference  :  74942
Health & Fitness  :  23298
Weather  :  52280
Utilities  :  18684
Travel  :  28244
Shopping  :  26920
News  :  21248
Navigation  :  86090
Lifestyle  :  16486
Entertainment  :  14030
Food & Drink  :  33334
Sports  :  23009
Book  :  39758
Finance  :  31468
Education  :  7004
Productivity  :  21028
Business  :  7491
Catalogs  :  4004
Medical  :  612


## Most Popular Genre Apps Available on  the Google Play Store

For the Google Play market, we actually have data on the number of installs, so we should be able to get a clearer picture of the popularity of genres. However, install numbers don't seem accurate enough, we can see that most of the values are open-ended (100 +, 1,000 +, 5,000 +, etc.):

In [25]:
display_table(free_google_apps, 5) # the Installs column

1,000,000+  :  15.73
100,000+  :  11.55
10,000,000+  :  10.55
10,000+  :  10.2
1,000+  :  8.39
100+  :  6.92
5,000,000+  :  6.83
500,000+  :  5.56
50,000+  :  4.77
5,000+  :  4.51
10+  :  3.54
500+  :  3.25
50,000,000+  :  2.3
100,000,000+  :  2.13
50+  :  1.92
5+  :  0.79
1+  :  0.51
500,000,000+  :  0.27
1,000,000,000+  :  0.23
0+  :  0.05
0  :  0.01


---

One problem with the data is that it's not accurate. For example, we don't know if an app with 100,000 + installs has 100,000 installs, 200,000 or 350,000. However, we don't need very accurate data for our purposes; we just want to get an idea of which app genres most users are attracted to, and we don't need perfect precision with respect to the number of users. 

We're going to leave the number as it is, which means we're going to consider that an app with 100,000 + installs has 100,000 installs.

However, in order to perform computations, we will need to convert each install number to float; this means that we need to remove commas and plus characters, otherwise the conversion will fail and cause an error. We'll do this in the coding cells below, where we'll also calculate the average number of installs for each genre(category).

---

In [23]:
#We are going to look at the unique app genre in Google Play Store
freq_table(free_google_apps,1)

{'ART_AND_DESIGN': 0.64,
 'AUTO_AND_VEHICLES': 0.93,
 'BEAUTY': 0.6,
 'BOOKS_AND_REFERENCE': 2.14,
 'BUSINESS': 4.59,
 'COMICS': 0.62,
 'COMMUNICATION': 3.24,
 'DATING': 1.86,
 'EDUCATION': 1.16,
 'ENTERTAINMENT': 0.96,
 'EVENTS': 0.71,
 'FINANCE': 3.7,
 'FOOD_AND_DRINK': 1.24,
 'HEALTH_AND_FITNESS': 3.08,
 'HOUSE_AND_HOME': 0.82,
 'LIBRARIES_AND_DEMO': 0.94,
 'LIFESTYLE': 3.9,
 'GAME': 9.72,
 'FAMILY': 18.91,
 'MEDICAL': 3.53,
 'SOCIAL': 2.66,
 'SHOPPING': 2.25,
 'PHOTOGRAPHY': 2.94,
 'SPORTS': 3.4,
 'TRAVEL_AND_LOCAL': 2.34,
 'TOOLS': 8.46,
 'PERSONALIZATION': 3.32,
 'PRODUCTIVITY': 3.89,
 'PARENTING': 0.65,
 'WEATHER': 0.8,
 'VIDEO_PLAYERS': 1.79,
 'NEWS_AND_MAGAZINES': 2.8,
 'MAPS_AND_NAVIGATION': 1.4}

In [24]:
# We are going to calcualte the average number of installs per app genre for the Google Play data set.
for key in freq_table(free_google_apps,1):
    total = 0
    len_category = 0
    for i in free_google_apps:
        category_app = i[1]
        if category_app == key:
            installs = i[5]
            installs = installs.replace('+','')
            installs = installs.replace(',','')
            installs = float(installs)
            total += installs
            len_category += 1
    avg_num_installs = total / len_category
    print(key, ' : ', round(avg_num_installs))

ART_AND_DESIGN  :  1986335
AUTO_AND_VEHICLES  :  647318
BEAUTY  :  513152
BOOKS_AND_REFERENCE  :  8767812
BUSINESS  :  1712290
COMICS  :  817657
COMMUNICATION  :  38456119
DATING  :  854029
EDUCATION  :  1833495
ENTERTAINMENT  :  11640706
EVENTS  :  253542
FINANCE  :  1387692
FOOD_AND_DRINK  :  1924898
HEALTH_AND_FITNESS  :  4188822
HOUSE_AND_HOME  :  1331541
LIBRARIES_AND_DEMO  :  638504
LIFESTYLE  :  1437816
GAME  :  15588016
FAMILY  :  3695642
MEDICAL  :  120551
SOCIAL  :  23253652
SHOPPING  :  7036877
PHOTOGRAPHY  :  17840110
SPORTS  :  3638640
TRAVEL_AND_LOCAL  :  13984078
TOOLS  :  10801391
PERSONALIZATION  :  5201483
PRODUCTIVITY  :  16787331
PARENTING  :  542604
WEATHER  :  5074486
VIDEO_PLAYERS  :  24727872
NEWS_AND_MAGAZINES  :  9549178
MAPS_AND_NAVIGATION  :  4056942


## Conclusions

In this project, we analyzed data on the Apple Store and the Google Play Store mobile apps with the aim of recommending an app profile that could be profitable for both markets.


We concluded that the a social and networking app could be profitable for both the Google Play and the Apple Store markets. The market is not over-saturated with social networking apps and only a few major apps, such as Facebook, Tinder, Whatsapp, Twitter and Snapchat, have high traction. These apps have their own special features so we could look at having a unique twist while still having the ability to do everything the other apps do with increased gamification.